assets/components/src/plugin-settings/SettingsSection.js

```diff
@@ -102,10 +100,10 @@ const SettingsSection = props => {
 			title={ title }
 			description={ description }
 			toggleChecked={ active }
-			hasGreyHeader={ active || hasGreyHeader }
+			hasGreyHeader={ hasGreyHeader }
```

Comment:
I wonder if we should have the `hasGreyHeader` property ever customizable since its state is basically determined by the state of other component's properties, which are processed internally (`active || active === null`).

Reply:
Oh, I see what you mean. Updated in 63ea672.